pkg/auth/acl/permission.go

```diff
@@ -59,12 +59,16 @@ func ACLToStatement(acl model.ACL) (model.Statements, error) {
 		if err != nil {
 			return nil, fmt.Errorf("%s: %w", acl.Permission, ErrBadACLPermission)
 		}
+		readConfigStatement, err := auth.MakeStatementForPolicyType("FSReadConfig", all)
+		if err != nil {
+			return nil, fmt.Errorf("%s: %w", acl.Permission, ErrBadACLPermission)
+		}
```

Comment:
I don't understand why we need this code.  ["FSRead" includes `fs:Read*`](https://github.com/treeverse/lakeFS/pull/5607/files#diff-1bfeff83ad028c5f61389248471de20c1d5708ca3a39c1c3e26f9d8970c043cfR59), why add ReadConfig explicitly?  (Is it because we now add `repositories/*` to the ARNs of FSRead?)

Reply:
It's added explicitly because ReadStorageConfig requires ReadConfig on `*` which fails on when scoping read to specific repositories